wip-0018.md

```diff
@@ -0,0 +1,84 @@
+<pre>
+  WIP: WIP-0018
+  Layer: Consensus (hard fork)
+  Title: Remove message argument from UnhandledIntercept RADON error
+  Authors: Tomasz Polaczyk <tomasz@witnet.foundation>
+  Discussions-To: `#dev-general` channel on Witnet Community's Discord server
+  Status: Draft
+  Type: Standards Track
+  Created: 2020-09-03
```

Comment:
Date is backdated 1 year (same as WIP-0017). LOL we're stuck in 2020. F*ck COVID.

Reply:
lol